site/zh-CN/userGuide/tools/insight_install.md

```diff
@@ -0,0 +1,29 @@
+---
+id: insight_install.md
+title: 安装 Milvus Insight
+---
+
+# 安装 Milvus Insight
+
+## 安装前提
+安装 Milvus Insight 前，请先确保你已安装[单机版 Milvus](https://milvus.io/docs/install_standalone-docker.md) 或[分布式版 Milvus](https://milvus.io/docs/install_cluster-docker.md)。
+
+<div class="alert note">
+Milvus Insight 仅支持 Milvus 2.0 或以上版本。
+</div>
+
+## 启动 Milvus Insight 实例
+
+```Apache
+docker run -p 8000:3000 -e HOST_URL=http://{ your machine IP }:8000 -e MILVUS_URL={your machine IP}:19530 milvusdb/milvus-insight:latest
```

Comment:
Just a proposal - in previous docs we use *<your_machine_ip>* kind of style.

Reply:
@shanghaikid Can we change the style as proposed above?